src/sonic-host-services-data/templates/sshd_config

```diff
@@ -0,0 +1,121 @@
+#	$OpenBSD: sshd_config,v 1.101 2017/03/14 07:19:07 djm Exp $
```

Comment:
Where is this file used? Is this added by mistake?

Reply:
yes, this was a mistake. removed.